doc/api/fs.md

```diff
@@ -47,9 +54,27 @@ try {
 }
 ```
 
-There is no guaranteed ordering when using asynchronous methods. So the
-following is prone to error because the `fs.stat()` operation may complete
-before the `fs.rename()` operation:
+## Promise Example
+
+`Promise`-based operations return a `Promise` that is resolved when the
+operation is complete.
+
+```js
+const fs = require('fs/promises');
+
+fs.unlink('/tmp/hello')
+  .then(() => console.log('successfully deleted /tmp/hello'))
+  .catch((error) => {
+    console.error('there was an error:', error.message);
+  });
```

Comment:
I’d be a fan of using async/await for all Promise API examples, unless it’s actually specific to Promises :)

Reply:
Yeah, I was wondering about that... should we go ahead and use top level await here or wrap it?

Reply:
I wouldn’t use TLA unless we also switch to `import` – wrapping it was what I had in mind, and it should be unproblematic given that we’d have a `try/catch` around the entire thing here :)